package.json

```diff
@@ -18,11 +20,16 @@
     {
       "name": "Nicolas Herment",
       "url": "https://github.com/nherment"
-    }
+    },
+    "jden <jason@denizac.org>"
   ],
-  "main": "index.js",
```

Comment:
no problem! for documentation's sake:

"main" in package.json sets the default package entry point, e.g. when other packages `require('radar')`. The default is `index.js`, so this change removes the redundant explicit declaration.